ui/src/app-bar/app-bar.tsx

```diff
@@ -25,6 +25,24 @@ const links: Link[] = [
 export const AppBar: React.FC = () => {
     const { t } = useTranslation();
     const [dataMenuOpen, setDataMenuOpen] = React.useState(false);
+    const dataMenuButtonInActiveStyle = `${styles['app-bar__link']} ${styles['app-bar__data-menu-button']}`;
+    const dataMenuButtonActiveStyle = `${styles['app-bar__link']} ${styles['app-bar__data-menu-button--active']}`;
+    const [dataMenuStyle, setDataMenuStyle] = React.useState<string>();
+    const [dataProductSelected, setDataProductSelected] = React.useState<boolean>(false);
+
+    React.useEffect(() => {
+        dataProductSelected
+            ? setDataMenuStyle(dataMenuButtonActiveStyle)
+            : setDataMenuStyle(dataMenuButtonInActiveStyle);
+    }, [dataProductSelected]);
+
+    const handleHeadingSelection = () => {
+        setDataProductSelected(false);
+    };
+
+    const handleDataProductSelection = () => {
+        setDataProductSelected(true);
+    };
```

Comment:
Jos luen tätä oikein, niin tämä taitaa olla eksplisiittisesti sidottu siihen, kun operaattori navigoi Tietotuotteisiin. Täten Selected taitaa olla false jos esim. refreshataan selain elementtilistauksessa 🤔 Tuon `activeStyle`n käytön voisi sitoa eksplisiittisten navigointien sijasta siihen, että löytyykö selaimen pathista `data-products`-merkkijono. Silloin myös nämä `useEffect`it ja erilliset `handleXyzSelection`it jäisivät täällä turhiksi

Reply:
Tuo on ehdottomasti parempi tapa :+1:

Reply:
Totta, tuossahan käy juuri niin että jos refreshaa niin tyylit katoavat. Kiitos siis hyvästä kommentista - korjaan!